index.html

```diff
@@ -11,5 +11,38 @@
     <title>My Portfolio2</title>
 </head>
 <body>
+  <div class="gen-container">
+    <nav>
+      <h3 class="logo">
+        My Logo
+      </h3>
+      <div class="menu-bars">
+        <img src="header-section\bars.svg" alt="">
+      </div>
+    </nav>
+    <div class="container"> 
```

Comment:
Kindly change this `div` into a `section` tag 🤗

- A section represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.